Decorator/TextDecorator/TextFilter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Decorator
+{
+    public abstract class TextFilter
```

Comment:
Probably it will be easier to implement with interface and not abstract class, but it's up to you